CAIPs/caip-69.md

```diff
@@ -0,0 +1,72 @@
+---
+caip: 69
+title: EOA extension to CAIP-10
+author: Griff Green (@griffgreen), Krati Jain (@jainkrati), Greg-the-Greek (Please edit Greg!)
+status: Draft
+type: Standard
+created: 2024-03-07
+---
+
+
+## Simple Summary
+
+CAIP-10 defines a way to identify an address in our multichain world using
+CAIP-2 blockchain id. CAIP-69 extends it to include a special case for EOAs
+using the chain ID of 0.
+
+
+## Abstract
+
+This proposal aims to facilitate specifying EOAs as multichain accounts
+extending CAIP-10 chain id specification. This is useful for both
+decentralized applications to communicate that EOAs can be used for multiple 
```

Comment:
Possibly you meant this: " Decentralized applications can use this to specify EOA as an address with chainID 0"

Reply:
Great improvement, will revise